etc/jetty/web.xml

```diff
@@ -20,6 +20,6 @@
 
 <web-app xmlns="http://java.sun.com/xml/ns/javaee" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
          xsi:schemaLocation="http://java.sun.com/xml/ns/javaee http://java.sun.com/xml/ns/javaee/web-app_3_0.xsd" version="3.0"
-         metadata-complete="true">
+         metadata-complete="false">
```

Comment:
Running an embedded jetty without this change will not fire up servlet context listeners that are annotated with @WebListener